src/pepr/operator/index.ts

```diff
@@ -40,3 +42,47 @@ When(UDSPackage)
   .Validate(validator)
   // Enqueue the package for processing
   .Reconcile(reconciler);
+
+//Watch for changes to the UDSExemption CRD and cleanup exemptions in policies Store
+When(UDSExemption).IsDeleted().Watch(removeExemptions);
+
+// Watch for changes to the UDSExemption CRD to enqueue an exemption for processing
+When(UDSExemption)
+  .IsCreatedOrUpdated()
+  .Validate(exemptValidator)
+  .Reconcile(async (exempt: UDSExemption) => {
+    if (!exempt.metadata?.namespace) {
+      Log.error(exempt, `Invalid Exemption definition`);
+      return;
+    }
+
+    const isPending = exempt.status?.phase === Phase.Pending;
+    const isCurrentGeneration = exempt.metadata?.generation === exempt.status?.observedGeneration;
+
+    if (isPending || isCurrentGeneration) {
+      Log.debug(exempt, `Skipping pending or completed exemption`);
+      return;
+    }
+
+    const { namespace, name } = exempt.metadata;
+
+    Log.debug(exempt, `Processing Exemption ${namespace}/${name}`);
+
+    try {
+      await updateStatus(exempt, { phase: Phase.Pending });
+
+      processExemptions(exempt);
+      await updateStatus(exempt, {
+        phase: Phase.Ready,
+        observedGeneration: exempt.metadata.generation,
+        titles: exempt.spec?.exemptions?.map(e => e.title || ""),
```

Comment:
Thinking about the titles here...if someone leaves them off it could give a weird impression on a status view that there are no exemptions included? Not sure if that's a reason to require titles, or try to auto-gen something based on the matcher? Information is obviously there regardless, just less clear if I'm looking at k9s for example.